armors/histerite-helmet.md

```diff
@@ -0,0 +1,21 @@
+# Histerite Helmet
+
+## Description
+Le casque de l'armure en Histerite.
+
+## Craft
+-=-
+ ==- https://raw.githubusercontent.com/ItamSurMc/histeria-wiki/main/.assets/items/emerald.png
+ ==- https://raw.githubusercontent.com/ItamSurMc/histeria-wiki/main/.assets/items/emerald.png
+ ==- 
+ ==- https://raw.githubusercontent.com/ItamSurMc/histeria-wiki/main/.assets/items/emerald.png
+ ==-
```

Comment:
Erreur de syntaxe de la balise

Reply:
Je ne vois pas où se situe l'erreur

Reply:
Il faut lire la documentation du wiki concernant les tables de craft et comment faire une cellule vide.
En résumé RTFM (Read The Fucking Manual)

Reply:
Fait partout